tests/client-dbus/tests/udev/_loopback.py

```diff
@@ -25,6 +25,9 @@
 
 _SIZE_OF_DEVICE = 1024 ** 4  # 1 TiB
 
+def _check_tokens(self, tokens):
```

Comment:
Move this into the ```LoopBackDevices``` class; that way you're using more standard object orientation.

Reply:
I was getting pylint errors when I first tried that (which probably means I was missing something):

```
./check.py tests
************* Module tests.udev._loopback
tests/udev/_loopback.py:87: [E0602(undefined-variable), LoopBackDevices.unplug] Undefined variable '_check_tokens'
tests/udev/_loopback.py:101: [E0602(undefined-variable), LoopBackDevices.generate_udev_add_events] Undefined variable '_check_tokens'
tests/udev/_loopback.py:121: [E0602(undefined-variable), LoopBackDevices.hotplug] Undefined variable '_check_tokens'
```

The above results if I try this modification to the file, as it is in the PR now:

```
diff --git a/tests/client-dbus/tests/udev/_loopback.py b/tests/client-dbus/tests/udev/_loopback.py
index 09322699..20e6869a 100644
--- a/tests/client-dbus/tests/udev/_loopback.py
+++ b/tests/client-dbus/tests/udev/_loopback.py
@@ -26,11 +26,6 @@ _LOSETUP_BIN = os.getenv("STRATIS_LOSETUP_BIN", "/usr/sbin/losetup")
 _SIZE_OF_DEVICE = 1024 ** 4  # 1 TiB
 
 
-def _check_tokens(self, tokens):
-    if not all(token in self.devices for token in tokens):
-        raise RuntimeError("No tokens found")
-
-
 class LoopBackDevices:
     """
     Class for creating and managing loop back devices which are needed for
@@ -46,6 +41,10 @@ class LoopBackDevices:
         self.count = 0
         self.devices = {}
 
+    def _check_tokens(self, tokens):
+        if not all(token in self.devices for token in tokens):
+            raise RuntimeError("No tokens found")
+
     def create_devices(self, number):
         """
         Create new loop back devices.
```

Reply:
You need to call it as an instance method: ```self._check_tokens(tokens)```.

Reply:
Aha; that works.  I was trying changing each call to `self._check_tokens(self, tokens)`, but that was getting "Too many positional arguments for method call".

Reply:
Resolved.